### ISSUE TITLE
Selected earthquake is not highlighted

### ISSUE THREAD

**Issue by boeckMt** [riesgos/dlr-riesgos-frontend#107]

**Comment by boeckMt** [comment]
The `olSelect` interaction triggers  -> `this.highlightedFeatures$.next(features)` which triggers -> `this.highlightedFeatures$.subscribe` -> `features.map(f => f.set('selected', true));` 

unfortunately the style is then not applied to the layer...

**Comment by boeckMt** [comment]
@MichaelLangbein I don't know how but the features in the layer `QuakeledgerProcess_selectedRows_selectedRows_result_layer` change somehow on `InteractionCompleted`. At least there `ol_uid` different and also the [`selected property` ](https://github.com/riesgos/dlr-riesgos-frontend/blob/master/src/app/components/map/map.component.ts#L231) is removed

**Comment by MichaelLangbein** [comment]
I'm wondering if we need that selection at all. In fact, I'm thinking of removing `highlightedFeatures$` from `map.component` completely. After all, an eq is only really selected after the user clicks 'send' on the selection step. Will investigate the removal of the `selected` property, though.

**Comment by boeckMt** [comment]
Yes, but at first the user should clearly see what have been chosen before execute the process.
If we allow the selection on the map, the the chosen one should be highlighted on the map as well.

I would also change the style of the selected earthquake later to something like this 

```ts
const props = feature.getProperties();
    const depth = props['origin.depth.value'];
    const magnitude = props['magnitude.mag.value'];
    let [r, g, b] = greenRedRange(6, 9, magnitude);
    const radius = linInterpolateXY(5, 10, 60, 5, depth);
    return [
      new olStyle({
        image: new olCircle({
            radius: radius + 4,
            fill: new olFill({
                color: [0, 114, 163, 0]
            }),
            stroke: new olStroke({
                color: [0, 114, 163, 0.5],
                width: 8
            })
        })
    }),
    new olStyle({
      image: new olCircle({
          radius: radius,
          fill: new olFill({
              color: [r, g, b, 0.5]
          }),
          stroke: new olStroke({
              color: [r, g, b, 1],
              width: 1
          })
      })
  })];
```

**Comment by boeckMt** [comment]
I would also write a description for the selection process, like on the `Earthquake catalogue`, then the user knows more easily what to do.


![image](https://user-images.githubusercontent.com/3969484/157630069-190e79a6-3cfa-4402-8722-2b611d207906.png)

E.g.


> Select a earthquake by choosing one of the dropdown. You may also select a earthquake by map, for this enable the map selection first.

FYI @eschoepfer

**Comment by MichaelLangbein** [comment]
Fixed with https://github.com/riesgos/dlr-riesgos-frontend/commit/31f540320c99a8ae49828582fc3e676c86c749c7
